src/components/Calculator.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const Calculator = () => (
+  <div className="grid-container">
+    <div className="row first">
+      <span className="full-span">0</span>
+    </div>
+    <div className="row second">
+      <span>AC</span>
+      <span>+/-</span>
+      <span>%</span>
+      <span>&divide;</span>
+    </div>
+    <div className="row third">
+      <span>7</span>
+      <span>8</span>
+      <span>9</span>
+      <span>x</span>
+    </div>
+    <div className="row fourth">
+      <span>4</span>
+      <span>5</span>
+      <span>6</span>
+      <span>-</span>
+    </div>
+    <div className="row fifth">
+      <span>1</span>
+      <span>2</span>
+      <span>3</span>
+      <span>+</span>
+    </div>
+    <div className="row sixth">
+      <span>0</span>
+      <span>.</span>
+      <span>=</span>
+    </div>
+  </div>
```

Comment:
- [ ] Based on the requirements,  your  Calculator.js should have at least 1 child component, it would be better to create a component  for your  calculator app inside Calculator.js, this will organize your UI into smaller, reusable pieces,  making your codebase more maintainable and easier to understand 👍🏿 

- [ ] It would be better for your design colors to match the wireframe, good job so far, for this milestone your design should match the wireframe 👍🏿  

Wireframe        |  Your design     
:-------------------------:|:-------------------------:
<img width="558" alt="calculator (1)" src="https://github.com/AliRezaBmeDu/math-magician/assets/69638013/26de315e-93ef-467e-812c-acb5887cc865"> |  ![Screenshot_140](https://github.com/AliRezaBmeDu/math-magician/assets/69638013/3ec1e27d-0e28-428c-b765-3e6c91d876c2)